test/EndToEndTests/Framework/Core/Client/DataServiceContextWrapper.cs

```diff
@@ -432,7 +432,7 @@ public bool Detach(object entity)
         /// <returns>batch response from which query results can be enumerated.</returns>
         public DataServiceResponse ExecuteBatch(DataServiceRequest[] param)
         {
-#if SILVERLIGHT || PORTABLELIB
+#if SILVERLIGHT || PORTABLELIB || (NETCOREAPP1_0 || NETCOREAPP2_0)

```

Comment:
>(NETCOREAPP1_0 || NETCOREAPP2_0) [](start = 34, length = 32)

why the extra parens?

Reply:
I'm anticipating that the NetCore macro will continue to follow the pattern NETCOREAPPX_X so I figure that putting parens would keep the || more organized.

Reply:
Will keep since it's a nit and I feel that it adds to future readability.